timon/scripts/http_json.py

```diff
@@ -26,15 +26,23 @@ def http_json(url, timeout=10, verify_ssl=True, cert=None):
         resp = requests.get(url, timeout=10, verify=verify_ssl, cert=cert)
     except Exception as exc:
         result['reason'] = repr(exc)
+        result['exit_code'] = flags.FLAG_ERROR
         return result
     s_code = resp.status_code
     result['status'] = s_code
-    if s_code == 200:
+    if s_code == 404:
```

Comment:
we might need a flag for the probes (in a future MR)
as in the case of some probes (e.g. the `NEW_ADM` probe) we'd like that a 404 will become `ERROR`

Reply:
yep